toplevel/role/role.go

```diff
@@ -221,6 +228,59 @@ func (c config) Apply(address string, entriesBytes []byte, dryRun bool, threadPo
 	return nil
 }
 
+// Extracts names of policies referenced within applicable attributes of desired roles
+// and updates attributes to only include the names.
```

Comment:
Apologies if I'm asking the obvious but what exactly is an attribute in the context of a role?

Reply:
potentially poor choice of term on my end. Attributes in this context are the properties within https://github.com/app-sre/qontract-schemas/blob/main/schemas/vault-config/role-1.yml .. updating comment to use `properties`